app/react/common/stacks/queries/useUpdateKubeGitStackMutation.ts

```diff
@@ -0,0 +1,53 @@
+import { useMutation } from 'react-query';
+
+import axios from '@/portainer/services/axios';
+import {
+  withError,
+  withInvalidate,
+  queryClient,
+} from '@/react-tools/react-query';
+import { StackId } from '@/react/common/stacks/types';
+import { EnvironmentId } from '@/react/portainer/environments/types';
+import { AutoUpdateModel } from '@/react/portainer/gitops/types';
+
+import { stacksQueryKeys } from './query-keys';
+import { buildStackUrl } from './buildUrl';
+
+type UpdateKubeGitStackPayload = {
+  AutoUpdate: AutoUpdateModel;
```

Comment:
can use small caps `camelCase` here

Reply:
The useUpdateKubeGitStackMutation function in this file turns out to have no references (not sure why I added it). I've moved the useUpdateKubeGitStackMutation function from `app/react/kubernetes/applications/components/RedeployGitAppForm/queries/useUpdateKubeGitStackMutation.ts` to this location.